.goreleaser.yaml

```diff
@@ -29,6 +29,16 @@ builds:
     goarch: *goarch
```

Comment:
PTAL at this comment: https://github.com/kubeshop/botkube-docs/pull/204#discussion_r1087549644